Apps/Sandcastle/gallery/LensFlare.html

```diff
@@ -0,0 +1,105 @@
+﻿<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1, minimum-scale=1, user-scalable=no">
+    <meta name="description" content="Lens Flare.">
```

Comment:
I'm noticing some distortion when resizing the screen in this branch, but not in 1.44:

![distortion](https://user-images.githubusercontent.com/915398/38814591-d463e930-415f-11e8-883e-922e08dec448.gif)